ql/src/test/org/apache/hadoop/hive/ql/metadata/TestHiveMetaStoreChecker.java

```diff
@@ -74,21 +76,21 @@
   @Before
   public void setUp() throws Exception {
     hive = Hive.get();
-    hive.getConf().setIntVar(HiveConf.ConfVars.METASTORE_FS_HANDLER_THREADS_COUNT, 15);
-    hive.getConf().set(HiveConf.ConfVars.HIVE_MSCK_PATH_VALIDATION.varname, "throw");
+    hive.getConf().set(MetastoreConf.ConfVars.FS_HANDLER_THREADS_COUNT.getVarname(), "15");
```

Comment:
setIntVar

Reply:
It does not work with MetasoreConf.ConfVars